extensions/wikia/Insights/InsightsController.class.php

```diff
@@ -85,9 +84,11 @@ private function renderSubpage() {
 	private function renderFlagsFiltering() {
 		if ( $this->model instanceof InsightsFlagsModel ) {
 			/* Enable flags filter in layout */
```

Comment:
As we're not using flagsFiltering anymore this comment is misleading. Can be removed.